Source/Views/TableViews/SPCopyTable.m

```diff
@@ -403,11 +403,11 @@ - (NSString *)rowsAsCsvStringWithHeaders:(BOOL)withHeaders onlySelectedRows:(BOO
  */
 - (NSString *)rowsAsSqlInsertsOnlySelectedRows:(BOOL)onlySelected{
 	
-	return [self rowsAsSqlInsertsOnlySelectedRows:onlySelected skipAutoIncrementColumn:NO];
+	return [self rowsAsSqlInsertsOnlySelectedRows:onlySelected skipAutoIncrementColumn:NO skipGeneratedColumn:NO];
```

Comment:
Why not skipGeneratedColumn:YES here?

Reply:
It was just to leave the previous behavior for other functions than "Copy as SQL insert" to avoir any issue.